ReactiveCocoa/ControlSubject.swift

```diff
@@ -0,0 +1,44 @@
+import ReactiveSwift
+import enum Result.NoError
+
+/// Represents a subject of an interactive control.
+///
+/// Generally speaking, `ControlSubject` emits changes only for the user
+/// interactions in which the subject specializes.
+public final class ControlSubject<Value> {
```

Comment:
@mdiep see my concerns w/ properties in UI controls here: https://github.com/ReactiveCocoa/ReactiveCocoa/issues/3229#issuecomment-251395000